From 97d505d247e486ef39e18f8c712c37b1b7622232 Mon Sep 17 00:00:00 2001
From: rmuxnet <armandas.kvietkus@proton.me>
Date: Tue, 7 Jul 2026 00:19:06 +0200
Subject: [PATCH] PCI: init dev=NULL in pci_scan_slot before Aeolia
 phantom-skip use
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------false"

This is a multi-part message in MIME format.
--------------false
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit

---
 drivers/pci/probe.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)


--------------false
Content-Type: text/x-patch; name="0140-PCI-init-dev-NULL-in-pci_scan_slot-before-Aeolia-pha.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0140-PCI-init-dev-NULL-in-pci_scan_slot-before-Aeolia-pha.patch"

diff --git a/drivers/pci/probe.c b/drivers/pci/probe.c
index cfe3ebf991258f8acdbdf4ebef8d7f5f49dd014d..6ca0a1ea3ad183f364f038e427dc8e0cdadf79c6 100644
--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -2873,7 +2873,7 @@ static int only_one_child(struct pci_bus *bus)
  */
 int pci_scan_slot(struct pci_bus *bus, int devfn)
 {
-	struct pci_dev *dev;
+	struct pci_dev *dev = NULL;
 	int fn = 0, nr = 0;
 
 	u32 l;

--------------false--


